examples/streetlights-operation-security.yml

```diff
@@ -47,9 +49,15 @@ channels:
       message:
         $ref: '#/components/messages/lightMeasured'
       security:
+      # This operation level security implies the ability to publish messages to 
+      # `smartylighting.streetlights.1.0.event.{streetlightId}.lighting.measured` channel with Authorization headers 
+      # that have `streetlights:write` scope. Note that an operation level security must still satisfy 
+      # security requirements specified at the server level.
        streetlights_auth:
-         - streetlights:write  
+        - streetlights:write  
```

Comment:
Security requirement object should be in the form of array, not object.

Reply:
Fixed